runtime/src/manifest.rs

```diff
@@ -117,10 +117,14 @@ pub(crate) fn load(
     match input {
         WasmInput::Data(data) => {
             let has_magic = data.len() >= 4 && data[0..4] == WASM_MAGIC;
-            let is_wat = data.starts_with(b"(module") || data.starts_with(b";;");
+            let s = std::str::from_utf8(&data);
+            let is_wat = s.is_ok_and(|data| {
+                let data = data.trim_start();
+                data.starts_with("(module") || data.starts_with(";;")
```

Comment:
(Ooh, this could also start with `(;`)

Reply:
ah thanks! i will open a new PR and link here. would `(;` be the start of a comment inside the opening paren?

Reply:
It's a block comment (there's an example [over here](https://github.com/dylibso/component-workbench/blob/main/00-core-module-greet.wat#L1-L6)!)

Reply:
also, would a paren followed by whitespace be valid? something like this:

```wat
(
module
```